draft-ietf-webtrans-http3.md

```diff
@@ -467,25 +467,13 @@ delivering application close metadata.
 
 \[\[RFC editor: please remove this section before publication.]]
 
-WebTransport over HTTP/3 uses two different mechanisms to negotiate versions for
-the different parts of the draft.
-
-The hop-by-hop wire format aspects of the protocol are negotiated by changing
-the codepoint used for the SETTINGS_ENABLE_WEBTRANSPORT parameter.  Because of
+The wire format aspects of the protocol are negotiated by changing the
+codepoint used for the SETTINGS_ENABLE_WEBTRANSPORT parameter.  Because of
```

Comment:
Is this something that would apply more generally, such that removal by the RFC editor isn't necessary?  This paragraph could probably be added to Section 3.2.  (The only bit that might be strange is a note about multiple webtransport versions, but that too is reasonable as a means of planning for the eventual replacement of this protocol.  That is, we could tweak it to say "A future specification may supersede this specification; that is, a future specification might require that this protocol be disabled if the setting for that protocol is included by both client and server."

(Maybe a separate PR is warranted for that change though...)

Reply:
If we were to make this scheme permanent, we should probably have a separate issue and discuss this with the WG.